### PR TITLE
Add docs for the I_S keyword table

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add documentation related to https://github.com/pingcap/tidb/pull/48807

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15795
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-keywords.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+字段含义如下：
+
+- `WORD`：关键字
+- `RESERVED`：关键字是否为保留关键字
+
+
+下面的 SQL 语句查询 `ADD` 和 `USER` 关键字的信息：
+
+```sql
+SELECT * FROM keywords WHERE WORD IN ('ADD','USER');
```
Comment (qiancai) [inline information-schema/information-schema-keywords.md:36]:
```suggestion
SELECT * FROM INFORMATION_SCHEMA.KEYWORDS WHERE WORD IN ('ADD','USER');
```

File: keywords.md [hunk 2]
```diff
@@ -56,6 +56,8 @@ CREATE TABLE test.select (BEGIN int, END int);
 Query OK, 0 rows affected (0.08 sec)
 ```
 
+TiDB 从 v7.6.0 开始提供 [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) 表，其中包含了所有 TiDB 关键字的信息。
```
Comment (qiancai) [inline keywords.md:59]:
```suggestion
TiDB 从 v7.6.0 开始提供 [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) 表，可以用于查询所有 TiDB 关键字。
```

File: information-schema/information-schema-keywords.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+2 rows in set (0.00 sec)
+```
+
+字段含义如下：
+
+- `WORD`：关键字
+- `RESERVED`：关键字是否为保留关键字
+
+
+下面的 SQL 语句查询 `ADD` 和 `USER` 关键字的信息：
```
Comment (qiancai) [inline information-schema/information-schema-keywords.md:33]:
```suggestion
例如，你可以使用以下 SQL 语句查询 `ADD` 和 `USER` 关键字的信息：
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: information-schema/information-schema-keywords.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+| RESERVED | int(11)      | YES  |      | NULL    |       |
++----------+--------------+------+------+---------+-------+
+2 rows in set (0.00 sec)
+```
+
+字段含义如下：
+
+- `WORD`：关键字
+- `RESERVED`：关键字是否为保留关键字
+
```
Comment (Oreoxmt) [inline information-schema/information-schema-keywords.md:31]:
```suggestio
```

**Pull request merged**